site/gatsby-site/content/about.mdx

```diff
@@ -49,32 +49,35 @@ The incident database is managed in a participatory manner by persons and organi
 **Non-Voting Members**
 
 * **[Neama Dadkhahnikoo](https://www.linkedin.com/in/neama/):** Neama Dadkhahnikoo is an expert in artificial intelligence and entrepreneurship, with over 15 years of experience in technology development at startups, non profits, and large companies. He currently serves as a Product Manager for Vertex AI, Google Cloud’s unified platform for end-to-end machine learning. Previously, Mr. Dadkhahnikoo was the Director of AI and Data Operations at the XPRIZE Foundation, CTO of CaregiversDirect (AI startup for home care), co-founder and COO of Textpert (AI startup for mental health), and a startup consultant. He started his career as a Software Developer for The Boeing Company. Mr. Dadkhahnikoo holds an MBA from UCLA Anderson; an MS in Project Management from Stevens Institute of Technology; and a BA in Applied Mathematics and Computer Science, with a minor in Physics, from UC Berkeley.  
-**Contributions:** Neama volunteers to keep the Responsible AI Collaborative organized outside his role as product manager at Google.
+**Contributions:** Neama serves as Executive Director to the Responsible AI Collaborative outside his role as product manager at Google.
 * **[Kit Harris](https://www.linkedin.com/in/kitharris/):** Kit leads on grant investigations and researches promising fields at Longview Philanthropy. He also writes about Longview recommendations and research for Longview client philanthropists. Prior to focusing on high-impact philanthropic work, Kit worked as a credit derivatives trader with J.P. Morgan. During that time, he donated the majority of his income to high-impact charities. Kit holds a first class degree in mathematics from the University of Oxford.  
 **Contributions:** Kit serves as board observer, provides strategic advice, and is RAIC's point of contact at Longview.
 
-
 # Collaborators
 
 **Open Source Contributors:** People that have contributed more than one pull request, graphics, site copy, or bug report to the AI Incident Database.
 
-* [César Varela](https://github.com/cesarvarela)
+* [César Varela](https://github.com/cesarvarela) is a Full Stack Engineer with the Responsible AI Collaborative
+* [Luna McNulty](https://lmcnulty.me/) is a UX Engineer with the Responsible AI Collaborative
 * [Alex Muscă](https://github.com/alexmcode)
```

Comment:
This looks good so far as I'm concerned. I'll note that César https://cesarvarela.com and Alex https://alexmusca.com both have websites. Did they ask to list their GitHub profiles instead?

Reply:
I don't think so, but I'll let everyone determine their own link.